src/features/encoders/mozJPEG/client/index.tsx

```diff
@@ -234,25 +268,87 @@ export class Options extends Component<Props, State> {
                   Smoothing:
                 </Range>
               </div>
-              <label class={style.optionTextFirst}>
+              <label
+                class={style.optionTextFirst}
+                title={getParameterDescription('quant_table')}
+              >
                 Quantization:
                 <Select
                   name="quant_table"
                   value={options.quant_table}
                   onChange={this.onChange}
                 >
-                  <option value="0">JPEG Annex K</option>
-                  <option value="1">Flat</option>
-                  <option value="2">MSSIM-tuned Kodak</option>
-                  <option value="3">ImageMagick</option>
-                  <option value="4">PSNR-HVS-M-tuned Kodak</option>
-                  <option value="5">Klein et al</option>
-                  <option value="6">Watson et al</option>
-                  <option value="7">Ahumada et al</option>
-                  <option value="8">Peterson et al</option>
+                  <option
+                    value="0"
+                    title={
+                      'Standard Quant Tables given in Annex K of the CCITT paper'
```

Comment:
Any reason why you put some descriptions in the meta file and some are inline? I think it’d be nice to move these into the meta file as well, so all descriptions for a codec are in one place.

Reply:
I wanna give descriptions for params _and_ their options (like `ImageMagick` for `quant_table`). Problem is, for intellisense to do some nice autocompletion for the `getParameterDescription` arguments, I'll need an interface containing all params and their options, which ends up looking very verbose IMO.

So I went with using `getParameterDescription` for the params, and inlining the descriptions for the options.

<hr>

I'll be refactoring all of this once I finish making a custom modal component (better explanations with better UI). For now I'll try to put all descriptions in the meta file; it'll definitely look cleaner once the modal is ready.